packages/cli/src/models/files/ZosVersion.ts

```diff
@@ -1,13 +1,42 @@
+import isNull from 'lodash.isnull';
 import isUndefined from 'lodash.isundefined';
 
-export const ZOS_VERSION = '2';
+const CURRENT_ZOS_VERSION = ['2', '2'];
+const [CURRENT_MAJOR_VERSION, CURRENT_MINOR_VERSION] = CURRENT_ZOS_VERSION;
+export const ZOS_VERSION = stringifyCurrentZosversion();
 
-export function checkVersion(version: string, where: any): boolean | never {
-  if (version === ZOS_VERSION) return true;
+export function checkVersion(version: string, where: any): void | never {
+  if (version === ZOS_VERSION) return;
   else if (isUndefined(version)) {
     throw Error(`zos version identifier not found in ${where}. This means the project was built with an older version of zos (1.x), and needs to be upgraded. Please refer to the documentation at https://docs.zeppelinos.org for more info.`);
-  }
-  else {
+  } else if (!isCurrentMajor(version) || (!isCurrentMinor(version) && !isUndefinedMinor(version))) {
     throw Error(`Unrecognized zos version identifier ${version} found in ${where}. This means the project was built with an unknown version of zos. Please refer to the documentation at https://docs.zeppelinos.org for more info.`);
   }
 }
+
+export function isNotMigratableZosversion(version: string): boolean {
```

Comment:
Since you always use it as `!isNotMigratableZosversion...`, I'd do 
```suggestion
export function isMigratableZosversion(version: string): boolean {
```